src/GraphQL/Authorization/GraphQLAuthorizeAttribute.cs

```diff
@@ -19,18 +26,52 @@ public GraphQLAuthorizeAttribute(string policy)
         /// <summary>
         /// The name of policy to apply.
         /// </summary>
-        public string Policy { get; }
+        public string? Policy { get; set; }
+
+        /// <summary>
+        /// A comma-separated list of the roles to apply.
```

Comment:
Are you sure that it is better than `string[]` ? Do you want to align this API completely to https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authorization.authorizeattribute.roles?view=aspnetcore-6.0#microsoft-aspnetcore-authorization-authorizeattribute-roles ?

Reply:
I want to align this API with the attribute that most ASP.NET Core developers are used to, where the name "Roles" (not "Role") and they use a comma-separated list to indicate multiple roles, any of which are acceptable to execute.

So, `[GraphQLAuthorize(Roles = "Administrator, Power User")]` should function just like `[Authorize(Roles = "Administrator, Power User")]` does (in concept).

Note that as it relates to the policy, the API is already compatible -- `[GraphQLAuthorize("MyPolicy")]` and `[GraphQLAuthorize(Policy = "MyPolicy")]` aligns with ASP.Net Core `[Authorize("MyPolicy")]` and `[Authorize(Policy = "MyPolicy")]`

Reply:
Just FYI, the authorize attribute has worked this way not only in ASP.Net Core since 1.0 but also in ASP.NET MVC 5.2 and likely prior versions as well.  So it is something people are likely used to, if they do any web programming in C#.

Reply:
C# allows to set string arrays when initializing attributes:
```csharp
 [GraphQLAuthorize("mypolicy", Roles = new[] { "myrole1", "myrole2" })]
 public class GraphQLAuthorizeAttribute : GraphQLAttribute
```

Reply:
OK. So someone has already thought about such a signature and we will use it.

Reply:
> I want to align this API with the attribute that most ASP.NET Core developers are used to

It's worth to add something like that into xml comments.

Reply:
>  So it is something people are likely used to, if they do any web programming in C#

This is a good argument.

Reply:
Added more notes and examples in the migration doc.